proto/viam/app/v1/app.proto

```diff
@@ -127,21 +145,59 @@ message ListOrganizationsResponse {
 }
 
 message Location {
+  // Location ID.
   string id = 1;
+  // Location name.
   string name = 2;
+  // Location creation timestamp.
   google.protobuf.Timestamp created_on = 3;
 }
 
 message CreateLocationRequest {
+  // Organization ID to create the location under.
   string organization_id = 1;
+  // Name of the location.
   string name = 2;
 }
 
 message CreateLocationResponse {
+  // Location object is returned.
   Location location = 1;
 }
 
+message GetLocationRequest {
+  // Location ID of location to get.
+  string location_id = 1;
+}
+
+message GetLocationResponse {
+  // Location object is returned.
+  Location location = 1;
+}
+
+message UpdateLocationRequest {
+  // Location ID of location to update.
+  string location_id = 1;
+  // The new to be updated on location.
+  optional string name = 2;
```

Comment:
I changed name to be `optional` which is different from what we reviewed in https://github.com/viamrobotics/api/pull/123/files#diff-e842ca0f50219c67d40be90a2a1efd40ee1b6df19a254357ab177d631b3ea4b4R313

I figure that during an update operation, most commonly either `name` or `parent_location_id` will be updated, but also updating both at the same time will be supported.

Reply:
👍